packages/web/src/pages/upload-page/fields/MultiTrackSidebar.module.css

```diff
@@ -0,0 +1,99 @@
+.root {
+  /* Required for Safari */
+  position: -webkit-sticky;
+  position: sticky;
+  top: 152px;
+  align-self: flex-start;
+  border-radius: 8px;
+
+  flex-shrink: 0;
+  width: 320px;
+  background: var(--neutral-light-10);
+  box-shadow: 0px 4px 8px 0px rgba(0, 0, 0, 0.06),
+    0px 0px 4px 0px rgba(0, 0, 0, 0.04);
+}
+
+.title {
+  padding: var(--unit-4);
+  border-radius: 8px 8px 0px 0px;
+  border-bottom: 1px solid var(--neutral-light-7);
```

Comment:
headsup i think their might be "border-strong" "border-default" colors that might slot in here